packages/did/src/Web3Names/web3names.chain.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2018-2021 BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import { SubmittableExtrinsic, IDidDetails } from '@kiltprotocol/types'
+import { BlockchainApiConnection } from '@kiltprotocol/chain-helpers'
+import { DecoderUtils } from '@kiltprotocol/utils'
+
+import type { Option, Bytes } from '@polkadot/types'
+
+import { DidUtils } from '../index.js'
+
+import { Web3NameOwner } from '../Did.chain.js'
+
+/**
+ *  Web3Name is the type of a nickname for a DID.
+ */
+type Web3Name = string
```

Comment:
M: If the type is used in exported functions, like this one is, then it should also be exported.

Reply:
fixed